catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -211,15 +178,54 @@ private void processContentItems(
     }
   }
 
+  private Map<String, Path> videoContentPaths(Map<String, Map<String, Path>> tmpContentPaths,
+      ContentItem contentItem) {
+    if (!isVideo(contentItem)) {
+      return null;
+    }
+
+    final long contentItemSizeBytes;
+    try {
+      contentItemSizeBytes = contentItem.getSize();
+    } catch (IOException e) {
+      LOGGER.warn(
```

Comment:
I know you didn't change any of the logging statements in this PR but I think these two `warn` logs should not be at the `warn` level. This one should probably be `debug`. The other one looks like it should be `info` or `debug`.